Packs/CommonScripts/Scripts/ParseYAML/ParseYAML.py

```diff
@@ -49,9 +49,9 @@ def yamlload(args: Dict[str, Any]) -> CommandResults:
     )
 
 
-def main():
+def main():  # pragma: no cover
```

Comment:
@samuelFain Why the no cover here?

Reply:
Since the logic is outside the main function i did to get more coverage, but because to current coverage for ParseYAML is already 100% i will remove it.

Reply:
putting resolve since Shmuel removed not cover